src/components/MessageBar/MessageBar.scss

```diff
@@ -13,11 +13,16 @@ $MessageBar-padding: 8px;
   @include ms-baseFont;
   padding: $MessageBar-padding;
   @include ms-bgColor-info;
+  color: $ms-color-infoText;
   width: 100%;
   box-sizing: border-box;
   display: flex;
   position: relative;
 
+  .ms-MessageBar-icon {
+    color: $ms-color-info;
```

Comment:
I wanted to set all the default colors in approximately the same place. This way the style of how we set the colors (default or otherwise) is more consistent.